Game crash with android custom build and using vibrate

**Godot version:**
3.2 RC3
**OS/device including version:**
Android 7
**Issue description:**
I am developping a game with godot 3.2 RC3. I use smartphone vibrate thank the function Input.vibrate_handheld(200).
I try to use the custom build. The export on my android phone is good.
But when the smartphone have to vibrate, the game crash.

**Steps to reproduce:**
Use the default custom build for android.
Make the smartphone vibrate.
**Minimal reproduction project:**

Tested on my Xiaomi Pocophone F1 running Android 9, I can confirm the bug.

On a test project using `Input.vibrate_handheld(300)`, I tested the following scenarios:

- Normal export (prebuilt template) with `VIBRATE` permission *disabled*: nothing happens, no error in logs (we should probably have an `ERR_FAIL` to make it clear to the user why it's not working)
- Normal export with `VIBRATE` permission *enabled*: works perfectly.
- Custom build (using Android source template and "Use Custom Build") with `VIBRATE` permission *enabled*: crashes with the following stacktrace:

```
1-25 18:33:59.889  1942  2070 E AndroidRuntime: FATAL EXCEPTION: GLThread 69234
01-25 18:33:59.889  1942  2070 E AndroidRuntime: Process: org.godotengine.bringoutyourdead, PID: 1942
01-25 18:33:59.889  1942  2070 E AndroidRuntime: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/support/v4/content/ContextCompat;
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at org.godotengine.godot.utils.PermissionsUtil.requestPermission(PermissionsUtil.java:79)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at org.godotengine.godot.Godot.requestPermission(Godot.java:1009)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at org.godotengine.godot.Godot.vibrate(Godot.java:365)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at org.godotengine.godot.GodotLib.step(Native Method)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at org.godotengine.godot.GodotRenderer.onDrawFrame(GodotRenderer.java:51)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1575)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1270)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.v4.content.ContextCompat" on path: DexPathList[[zip file "/data/app/org.godotengine.bringoutyourdead-g3gi68psd1xd3b_YG9kvLg==/base.apk"],nativeLibraryDirectories=[/data/app/org.godotengine.bringoutyourdead-g3gi68psd1xd3b_YG9kvLg==/lib/arm64, /data/app/org.godotengine.bringoutyourdead-g3gi68psd1xd3b_YG9kvLg==/base.apk!/lib/arm64-v8a, /system/lib64, /vendor/lib64]]
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:171)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
01-25 18:33:59.889  1942  2070 E AndroidRuntime: 	... 7 more
```

--- a/platform/android/java/app/build.gradle
+++ b/platform/android/java/app/build.gradle
@@ -33,7 +33,7 @@ allprojects {
 }
 
 dependencies {
-	implementation libraries.supportCoreUtils
+    implementation libraries.supportCoreUtils
 
     if (rootProject.findProject(":lib")) {
         implementation project(":lib")

--- a/platform/android/java/app/build.gradle
+++ b/platform/android/java/app/build.gradle
@@ -33,6 +33,8 @@ allprojects {
 }
 
 dependencies {
+	implementation libraries.supportCoreUtils
+
     if (rootProject.findProject(":lib")) {
         implementation project(":lib")
     } else {

--- a/platform/android/java/app/config.gradle
+++ b/platform/android/java/app/config.gradle
@@ -4,11 +4,13 @@ ext.versions = [
         minSdk : 18,
         targetSdk : 28,
         buildTools : '28.0.3',
+		supportCoreUtils : '28.0.0'
 
 ]
 
 ext.libraries = [
-        androidGradlePlugin : "com.android.tools.build:gradle:$versions.androidGradlePlugin"
+        androidGradlePlugin : "com.android.tools.build:gradle:$versions.androidGradlePlugin",
+		supportCoreUtils : "com.android.support:support-core-utils:$versions.supportCoreUtils"
 ]
 
 ext.getExportPackageName = { ->

--- a/platform/android/java/app/config.gradle
+++ b/platform/android/java/app/config.gradle
@@ -4,13 +4,13 @@ ext.versions = [
         minSdk : 18,
         targetSdk : 28,
         buildTools : '28.0.3',
-		supportCoreUtils : '28.0.0'
+        supportCoreUtils : '28.0.0'
 
 ]
 
 ext.libraries = [
         androidGradlePlugin : "com.android.tools.build:gradle:$versions.androidGradlePlugin",
-		supportCoreUtils : "com.android.support:support-core-utils:$versions.supportCoreUtils"
+        supportCoreUtils : "com.android.support:support-core-utils:$versions.supportCoreUtils"
 ]
 
 ext.getExportPackageName = { ->

--- a/platform/android/java/lib/build.gradle
+++ b/platform/android/java/lib/build.gradle
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.library'
 
 dependencies {
-	implementation libraries.supportCoreUtils
+    implementation libraries.supportCoreUtils
 }
 
 def pathToRootDir = "../../../../"

--- a/platform/android/java/lib/build.gradle
+++ b/platform/android/java/lib/build.gradle
@@ -1,7 +1,7 @@
 apply plugin: 'com.android.library'
 
 dependencies {
-    implementation "com.android.support:support-core-utils:28.0.0"
+	implementation libraries.supportCoreUtils
 }
 
 def pathToRootDir = "../../../../"

